### ISSUE TITLE
Ensure validation errors in file picker forms are labeled correctly

### ISSUE THREAD

**Issue by robertknight** [hypothesis/lms#6012]
From the [2024 VPAT review](https://docs.google.com/document/d/1YdYr5jIsKEm4CkOQF9un3LUc-A8xLtwFP5kWbNpqYOM/edit#heading=h.gjdgxs), we have validation errors in the file picker UI (and possibly other LMS frontend UIs) which are not announced to assistive technology and/or not connected to the associated control properly.

See the section that begins:

> Errors for the URL input within the assignment creation dialog are not announced to assistive technology users and are not programmatically associated with the input.
